pkgs/development/python-modules/ansible/collections.nix

```diff
@@ -14,21 +14,23 @@
 , xmltodict
 , withJunos ? false
 , withNetbox ? false
+
+, version ? null
+, sha256 ? null
```

Comment:
I don't think we want the ``? null`` here. If the values are not supplied it should error.